### ISSUE TITLE
Meshes do not draw when within the far clip range and scaled

### ISSUE THREAD

**Issue by AMZN-alexpete** [o3de/o3de#9375]
**Describe the bug**
When a mesh is scaled up and moved far away - but still within the far-clip range for the camera, the mesh will suddenly stop drawing.  This is likely because of a miscalculation in the pixel coverage in the visual/occlusion system.

**Steps to reproduce**
1. Create a new level
2. Set the camera far plane to 5000 for the default camera in the scene
3. Choose 'be this camera' so you can see the world with that camera's settings
4. Orient the camera so it's looking at the shader ball
5. Scale the shader ball
6. Move the shader ball away from the camera, still within the far clip range, until it disappears

**Expected behavior**
The shader ball keeps drawing until it reaches the far clip, or the pixel coverage of the shaderball mesh is less than 1.

**Actual behavior**
The shader ball suddenly stops drawing when it is still very large on screen and easily within the farclip range.

**Assets required**
n/a

**Screenshots/Video**
Shader ball (not even scaled) and visible at 1372m
![image](https://user-images.githubusercontent.com/26804013/166605934-d8b9072b-e13d-4ed4-802e-1aced799e3bb.png)
Shader ball no longer visible at 1373m
![image](https://user-images.githubusercontent.com/26804013/166605940-f4773199-1fee-43de-a2b3-2ecb9d8fc43b.png)

**Found in Branch**
development/stab

**Desktop/Device (please complete the following information):**
 - Device: PC
 - OS: Windows
 - Version 10

**Comment by AMZN-alexpete** [comment]
I have found that updating the lod selection radius in `MeshFeatureProcessor.cpp` on line 1306 seems to fix the issue
```
m_cullable.m_lodData.m_lodSelectionRadius = 0.5f*localAabb.GetExtents().GetMaxElement();
```

I'll try to get a PR created with an updated unit test.

**Comment by AMZN-alexpete** [comment]
A fix has been merged in PR #9649
